addon-test-support/@ember/test-helpers/settled.js

```diff
@@ -146,7 +143,7 @@ export default function settled(options) {
         timeout = MAX_TIMEOUT;
       }
 
-      global.setTimeout(function() {
+      nextTick(function() {
```

Comment:
since this invocation is called with a non-0 timeout value does it make sense to call something called `nextTick` here?

Reply:
I can export it with a different name. I agree it isn’t perfect. 

The basic gist that we need to do is ensure we use the eagerly cached `setTimeout` (which is what `nextTick` actually is).

Reply:
sure, I understand why it's there and why it's used, I'm just not sure it is the best name. I like the name for the situation where you don't pass any timeout at all, but once you pass a timeout the name is not fitting anymore since it won't actually be the *next* tick 😉

Reply:
Yep, totally agree. I’ll export it as `futureTick` also 😜 and use that here...

Reply:
👍 - Updated now.